components/app-search.tsx

```
@@ -88,43 +99,72 @@ export function AppSearch({ onViewChange }: AppSearchProps) {
-        return apps.filter(app => 
-          utilityKeywords.some(keyword => 
-            app.title.toLowerCase().includes(keyword)
-          )
+        const utilityKeywords = [
+          "calculator",
+          "terminal",
+          "console",
+          "system",
+          "preferences",
+          "utility",
+        ];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)